controllers/operator/configmap.go

```diff
@@ -939,10 +939,10 @@ func getClusterAddress(client client.Client, namespace string, configMap string)
 	currentConfigMap := &corev1.ConfigMap{}
 	err := client.Get(context.TODO(), types.NamespacedName{Name: configMap, Namespace: namespace}, currentConfigMap)
 	if err != nil {
-		log.Info("Error getting configmap", configMap)
+		log.Info("Error getting configmap", "ConfigMap.Name", configMap, "ConfigMap.Namespace", namespace)
 	} else {
 		host := currentConfigMap.Data["cluster_address"]
-		log.Info("Fetched cluster address from configmap", host)
+		log.Info("Fetched cluster address from configmap", " with cluster_address value ", host)
```

Comment:
```suggestion
		log.Info("Fetched cluster address from configmap", "cluster_address", host)
```

Reply:
Someone should be able to use a tool like jq to grab the specific kvp easily. Or filter in some monitoring service that's consuming logs.